addons/mail/static/src/components/follower_list_menu/follower_list_menu.xml

```diff
@@ -12,13 +12,13 @@
 
                     <t t-if="state.isDropdownOpen">
                         <div class="o_FollowerListMenu_dropdown dropdown-menu dropdown-menu-right" role="menu">
-                            <t t-if="thread.channel_type !== 'chat'">
+                            <t t-if="thread.channel_type !== 'chat' and thread.hasWriteAccess">
```

Comment:
This is old code but while I see it, followers is no longer support for channels at all, so `thread.channel_type !== 'chat'` should be `thread.model !== 'channel'`